basictracer/context.py

```diff
@@ -0,0 +1,7 @@
+class Context(object):
```

Comment:
this could use a docstring explaining the range of values for IDs, and their relationships